PotreeConverter/include/LASPointReader.h

```diff
@@ -113,7 +113,7 @@ class LIBLASReader{
 
         Point p = transform(coordinates[0], coordinates[1], coordinates[2]);
         p.intensity = point->intensity;
-        p.classification = point->classification;
+		p.classification = (point->classification ? point->classification : point->extended_classification);
```

Comment:
@m-schuetz this seems to be assuming that point->extended_classification will be present if point->classification is not. I don't think that's a good assumption.